### PR TITLE
Ci/fix-docker-names

### PR DESCRIPTION
**Pull request by desimone**
fix makefile to use version package
try go releaser

Signed-off-by: Bobby DeSimone <bobbydesimone@gmail.com>

### CODE REVIEW

**Comment by calebdoxsey** [comment]
This no longer publishes for PRs.

File: .github/workflows/release-tagged.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - name: Get tag name
+        id: tagName
+        run: |
+          TAG=$(git describe --tags --exact-match)
+          echo ::set-output name=tag::${TAG}
+          echo ::set-output name=version::${TAG#v}
+
+      - name: Install Cloudsmith CLI
+        run: |
+          pip3 install cloudsmith-cli
```
Comment (wasaga) [inline .github/workflows/release-tagged.yml:55]: is that required?

**Review by wasaga — APPROVED** [review]

File: .github/workflows/release-tagged.yml (hunk 1 above)
Comment (desimone) [reply .github/workflows/release-tagged.yml:55]: Yes, I forgot rpm/dpm images.

**Review by calebdoxsey — APPROVED** [review]
I didn't really review this. I don't know what most of this config does.

**Pull request merged**